python/cudf/cudf/tests/test_column.py

```diff
@@ -398,8 +398,8 @@ def test_column_view_string_slice(slc):
             cudf.core.column.as_column([], dtype="uint8"),
         ),
         (
-            cp.array([453], dtype="uint8"),
-            cudf.core.column.as_column([453], dtype="uint8"),
+            cp.array([255], dtype="uint8"),
```

Comment:
I guess it doesn't matter what value we choose here? Just wondering if it's important to use 453-256.

Reply:
I don't think it matters. I use `255` just because it's `np.iinfo(uint8).max'